### ISSUE TITLE
Automate the release procedure

### ISSUE THREAD

**Issue by alansemenov** [enonic/cli-enonic#314]
* Should be done with Github actions, like in other repos
* `ARTIFACTORY_PASSWORD` should be taken from Github like in other repos, not [environment variable](https://github.com/enonic/cli-enonic/blob/master/README.md#publishing)
* Publishing to Snapcraft requires logging with login/password in the terminal before starting the release. Those should also be stored somewhere and fetched by the release procedure.

**Comment by alansemenov** [comment]
Duplicate of https://github.com/enonic/cli-enonic/issues/314
